ui/app/components/file-to-array-buffer.js

```diff
@@ -29,7 +29,9 @@ export default Component.extend({
 
   readFile(file) {
     const reader = new FileReader();
-    reader.onload = () => this.send('onChange', reader.result, file);
+    if (!this.isDestroyed && !this.isDestroying) {
```

Comment:
Can we get a little info on why this was added, for posterity?

Reply:
Sure thing I added a couple of comments.